### ISSUE TITLE
Archiving this feedstock?

### ISSUE THREAD

**Issue by jakirkham** [conda-forge/openmm-atmmetaforce-plugin-feedstock#17]
Recently the contents of this plugin were upstreamed to OpenMM in PR ( https://github.com/openmm/openmm/pull/4110 ), which has been included in the OpenMM 8.1.0 release ( recently published on conda-forge: https://github.com/conda-forge/openmm-feedstock/pull/120 )

Given this, would it make sense to archive this feedstock?

**Comment by egallicc** [comment]
Yes, it makes sense to archive the openmm-atmmetaforce-plugin feedstock. Its functionality is reproduced by OpenMM 8.1.0's ATMForce().

**Comment by jakirkham** [comment]
Thanks Emilio! 🙏

Submitted an archive request with PR: https://github.com/conda-forge/admin-requests/pull/868
